walsamlee/politico
# #163416770 Add View a Specific Party Endpoint

#### What does this PR do?
Add `view-specific-party` API endpoint
#### Description of Task to be completed?
Have `/api/v1/parties/:partyId` endpoint working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-view-party-163416770`
- Using postman, `GET` with API endpoint URL `/api/v1/parties/:partyId` 
#### What are the relevant pivotal tracker stories?
#163416770
